refactor/urls.js

```diff
@@ -1,16 +1,19 @@
-// Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+    if (tasks?.prevTasks) {
+        return { url: '/tasks?hasPrev=true' };
```

Comment:
Feels wrong. Please compare the logic again

Reply:
I thought that in original code the below line was intentional, for overriding all checks up until that point... wasn't it the case?

This line: `url = '/tasks?hasPrev=true';`

@ankushdharkar